01/index.html

```diff
@@ -8,6 +8,6 @@
     <title>devmentor.pl - JS Tools - #01</title>
 </head>
 <body>
-    <p>GitHub też jest fajny!</p>
+    <h1>Git jest fajny!</h1>
```

Comment:
👍 

Początkujące osoby często mają problem z praca nad kopią repozytorium (forkiem), a nie wersją moją.
Jeśli starają się coś spushować na GH to nie mają uprawnień, aby to zrobić - wtedy jest błąd w terminu.
Ponieważ często nie czytają błędów to nie wiedzą co z tym zrobić.
Najprościej jest sprawdzić jakie adresy mają zapisane w konfiguracji (`git remote -v`) - jeśli nie ma tam loginu to to oznacza, że przy klonowaniu repo wykorzystali link do mojego repo, a nie swojego. Można to zmienić przy pomocy `git remote set-url` -> https://linuxhint.com/change-git-remote-origin-url/